src/frontends/tensorflow/src/utils.hpp

```diff
@@ -174,9 +174,9 @@ void values_from_const_node(const NodeContext& node, ov::Shape* const_tensor_sha
                                   "handle this element type";
                 FRONT_END_THROW("Encountered unknown element type " + DataType_Name(dt) + " on an empty tensor_proto");
             }
-            TENSORFLOW_OP_VALIDATION(node, val_size != 0, "Empty values vector");
-
-            if (i < val_size) {
+            if (val_size == 0) {
```

Comment:
what is a type of `Const`? Should we update with a new type that switch? And for that new type we should retrieve default_value indicated in `Const` tensor

Reply:
type is int32, no update to switch needed.

Reply:
Why wasn't it handled then on line 147 for which `val_size` is not zero?